src/shared/enums/order-status.ts

```diff
@@ -0,0 +1,6 @@
+export enum OrderStatus {
+  processing = 'Processing',
+  underDelivery = 'Under delivery',
+  completed = 'Completed',
+  cancelled = 'Cancelled',
```

Comment:
Order status shouldn't have the translated text.
It's better to save here some keys, like the property names that you have here,
and you can use these keys in i18n and get the translation from there.